### ISSUE TITLE
Installing on Fusion June/2023

### ISSUE THREAD

**Issue by Shoruk007** [martinhansdk/Flatpack#2]
After attempting to install, I fail to find the pulldown for the addin under Utilities>Make submenu?
After Running the Addin nothing seems to happen - Wondering if the update to Fusion it not allowing the plugin to work.

**Comment by martinhansdk** [comment]
Yes, the addon needs an update because the menu structure changed. Autodesk has made it harder to publish these addons. I made an effort to comply with their current procedures a couple of years ago, but it was rejected in the review process because they wanted me to build the installer in a different way. I didn't manage to do that then and haven't picked up the project since. So no, currently the add-on doesn't work, unfortunately.
